src/controllers/reactWebviewBaseController.ts

```diff
@@ -371,6 +371,11 @@ export abstract class ReactWebviewBaseController<State, Reducers>
         this._onDisposed.fire();
         this._disposables.forEach((d) => d.dispose());
         this._isDisposed = true;
+        this.extraDispose();
```

Comment:
Override the dispose method in your derived class and
public dispose() {
	super.dispose();
	// do stuff
}

Reply:
updated